src/snowflake/snowpark/modin/plugin/docstrings/base.py

```diff
@@ -2048,6 +2048,90 @@ def reorder_levels():
     def resample():
         """
         Resample time-series data.
+
+        Convenience method for frequency conversion and resampling of time series.
+        The object must have a datetime-like index (DatetimeIndex, PeriodIndex, or TimedeltaIndex),
+        or the caller must pass the label of a datetime-like series/index to the ``on``/``level`` keyword parameter.
+
+        Parameters
+        ----------
+        rule : DateOffset, Timedelta or str
+            The offset string or object representing target conversion.
+        axis : {0 or 'index', 1 or 'columns'}, default 0
+            Which axis to use for up- or down-sampling. For Series this parameter is unused and defaults to 0.
+            Must be DatetimeIndex, TimedeltaIndex or PeriodIndex.
+
+            Deprecated since version 2.0.0: Use frame.T.resample(…) instead.
+        closed : {'right', 'left'}, default None
+            Which side of bin interval is closed. The default is 'left' for all frequency offsets except for
+            'ME', 'YE', 'QE', 'BME', 'BA', 'BQE', and 'W' which all have a default of 'right'.
```

Comment:
do we support all of these values? same question for other parameters

Reply:
This is straight from pandas docs, put what we support in the Notes section. Is that not how it should be done?